JobConfig/cosmic/S1DSStops.fcl

```diff
@@ -50,7 +50,7 @@ physics.end_paths : [ EndPath ]
 # no histogram output
 services.TFileService.fileName : "/dev/null"
 services.SeedService.baseSeed         :  773651
-services.SeedService.maxUniqueEngines :  2
+services.SeedService.maxUniqueEngines :  3
```

Comment:
We agreed today to set maxUniqueEngines in standardServices.  That PR is under review, after it is approved you should remove this line.